index.html

```diff
@@ -63,9 +57,8 @@ <h3>I'm a Software Developer</h3>
         my work and experience! If you like what you see and have a project you
         need coded, don't hesitate to contact me.
       </p>
-
       <footer>
-        <div class="social-links d-flex justify-content-center">
+        <div class="social-links">
```

Comment:
I think you need to align the icon to the left as shown in figma.
And the gap between text and icon is too far. Please make sure the gap between text and icon to be the same.